### ISSUE TITLE
Implement `cupy.cuda.cutensor.get_build_version`

### ISSUE THREAD

**Issue by kmaehashi** [cupy/cupy#3351]
`cupy.cuda.cudnn.get_build_version()` equivalent for cuTENSOR.

**Comment by emcastillo** [comment]
ITs done in https://github.com/cupy/cupy/pull/3271 :)

**Comment by kmaehashi** [comment]
The version used in the build time and runtime may be different 😅

**Comment by emcastillo** [comment]
Good point 😅

**Comment by leofang** [comment]
> The version used in the build time and runtime may be different 😅

Right, this would be useful (in the future) for the conda package. I raised a few issues to support cuTENSOR in conda-forge: https://github.com/conda-forge/cupy-feedstock/issues/57, https://github.com/conda-forge/docker-images/issues/137

**Comment by emcastillo** [comment]
I wonder if we should create some file with the config used when building

**Comment by leofang** [comment]
This would be nice! If you do, please distribute the file in the wheel/sdist/whatnot.

**Comment by leofang** [comment]
(and we can do the same thing in the conda package)

**Comment by emcastillo** [comment]
We discussed last week that it may be better to use a define passed to the compiler as it is done with cudnn :D

**Comment by leofang** [comment]
Yeah that's also fine I think.
